tensorboard/components/experimental/plugin_util/core-host-impl.ts

```diff
@@ -15,12 +15,12 @@ limitations under the License.
 /**
  * Implements core plugin APIs.
  */
+import '../../../webapp/tb_polymer_interop_types';
```

Comment:
PR description hygiene: You include a couple screenshots that IMO are not very valuable. I think you're trying to tell me how you tested your changes but you could probably just use words for that.

A more valuable screenshot would be one that actually shows how the new code is resilient to variable renaming (perhaps a similar screenshot to the one I posted where you see how window.tensorboard.tf_globals appears in the console).

Reply:
It didn't load with property renaming enabled